Scripts/CI/README_Metadata_StyleCheck/entry.py

```diff
@@ -0,0 +1,564 @@
+import os
+import argparse
+import subprocess as sp
+from os import listdir
+from os.path import isfile, join
+import json
+
+def main():
+    # Tell the program that we're using global variables
+    msg = 'Entry point of the docker to run json style check scripts.'
+    parser = argparse.ArgumentParser(description=msg)
+    parser.add_argument('-s', '--string', help='A JSON array of file paths.')
+    args = parser.parse_args()
+    cleanstring = ""
+    try:
+        for letter in args.string:
+            if letter not in '[" \']':
+                cleanstring+=letter
+        file_list = cleanstring.split(",")
+    except Exception as e:
+        print(f"Unable to split args {args.string}.\nException: {e}")
+        exit(1)
+    check_files(file_list)
+    
+def check_files(file_list):
+    print("Files to check: ")
+    for file in file_list:
+        print(file)
+    print("\n")
+    total_errors = 0
+    if not file_list:
+        print("No files in file set")
+        exit(0)
+    for file in file_list:
+        try:
+            errors = []
+            # Set variables for file
+            directory_list = file.split("/")
+            file_name = directory_list[-1]
+            
+            # Check skip file conditions
+            # Does the file exist, is it in a category, is it a .json or .md?
+            if skip_file(directory_list):
+                continue        
+
+            # If metadata file
+            if file_name.lower() == 'readme.metadata.json':
+                try:
+                    json_file = MetadataFile(file)
+                    errors += json_file.check_metadata_file_for_errors()
+                except Exception as e:
+                    errors.append(f"Error checking {file_name}. Exception: {e}")
+
+            # If README.md
+            elif file_name.lower() == 'readme.md':
```

Comment:
Good check! 👍 

I assume `file_name.lower()` converts it to all lowercase. This is a good check since we try to keep all new samples with it in all capitals but there are some exceptions with existing samples.